src/Playwright/Core/BrowserType.cs

```diff
@@ -77,23 +77,21 @@ public async Task<IBrowserContext> LaunchPersistentContextAsync(string userDataD
             options ??= new BrowserTypeLaunchPersistentContextOptions();
             return (await _channel.LaunchPersistentContextAsync(
                 userDataDir,
-                args: options.Args,
```

Comment:
Curious why this shuffled

Reply:
IIRC this was me, applying a refactoring, re-ordering the parameters to be in the same order as in the interface.